cli/src/commands/squash.rs

```diff
@@ -154,6 +154,10 @@ pub(crate) struct SquashArgs {
     #[arg(long, short, conflicts_with = "message_paragraphs")]
     use_destination_message: bool,
 
+    /// Don't open an editor to edit the squashed commit's description
```

Comment:
I'm not sure if it would be better to say why this is useful. Maybe something like

```suggestion
    /// Don't open an editor to edit the squashed commit's description. Instead, all squashed commits descriptions will be combined.
```